packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -426,6 +446,14 @@ export const DEFAULT_INTERMEDIATE_TOKENS_BY_CHAIN_ID = valueByChainId<string[]>(
             '0xad6d458402f60fd3bd25163575031acdce07538d', // DAI
             '0x07865c6e87b9f70255377e024ace6630c1eaa37f', // USDC
         ],
+        [ChainId.Polygon]: [
+            getContractAddressesForChainOrThrow(ChainId.Polygon).etherToken,
+            '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619', // WETH
```

Comment:
I'd re-use `POLYGON_TOKENS` since its been defined. Should've done it for BSC.

DAI address here is mainnet DAI?

Reply:
👍  whoops, I must've copied from the left column instead of the right one on https://mapper.matic.today 😂